worlds/stardew_valley/regions.py

```diff
@@ -353,10 +314,32 @@ def inverted(self):
 ]
 
 
+def create_final_regions(world_options: StardewOptions) -> List[RegionData]:
+    vanilla_regions = stardew_valley_regions
+    if world_options[options.Mods] is None:
+        return vanilla_regions
+    for mod in world_options[options.Mods]:
+        if mod not in ModDataList:
+            continue
+        for mod_region in ModDataList[mod].regions:
+            existing_region = next(
+                (region for region in vanilla_regions if region.name == mod_region.name), None)
+            if existing_region:
+                if all(exits in existing_region.exits for exits in mod_region.exits):
+                    continue
+                existing_region.exits.extend(mod_region.exits)
```

Comment:
If all of the mod exists already are there, do nothing
otherwise, add all of the mod exits

What if only part of the mod exits are already there? This will add them all, so the ones that were already there will be duped?